lib/embedded_ansible/appliance_embedded_ansible.rb

```diff
@@ -70,6 +71,32 @@ def api_connection
     api_connection_raw("localhost", HTTP_PORT)
   end
 
+  def create_local_playbook_repo
+    self.class.consolidate_plugin_playbooks(playbook_repo_path)
+
+    Dir.chdir(playbook_repo_path) do
+      require 'rugged'
+      repo = Rugged::Repository.init_at(".")
+      index = repo.index
+      index.add_all("*")
+      index.write
+
+      options              = {}
+      options[:tree]       = index.write_tree(repo)
+      options[:author]     = options[:committer] = { :email => "system@localhost", :name => "System", :time => Time.now.utc }
+      options[:message]    = "Initial Commit"
+      options[:parents]    = []
+      options[:update_ref] = 'HEAD'
+      Rugged::Commit.create(repo, options)
+    end
+
+    FileUtils.chown_R('awx', 'awx', playbook_repo_path)
```

Comment:
Why does awx own the repo?  I know that was there before, but that doesn't make any sense to me.

Reply:
The assumption was the awx user would be reading this git repo's path when we posted this local path for the new project.  If it has permission to read in all situations, then this can be removed.